site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -42,7 +41,7 @@ field. This field uses the `"service_worker"` key, which specifies a single Java
   "name": "Awesome Test Extension",
   ...
   "background": {
-    "service_worker": "background.js"
+    "service_worker": "service-worker.js"
```

Comment:
At the moment the canonical example filename for the main background entry point is `bacgkround.js`. This practice was established at some point in he past and carried over to Manifest V3. Do you want to change this across all our docs and samples?

Reply:
I actually do. I've observed conversations on StackOverflow where one party was talking about V2 and the other was talking about V3, but they didn't know it because they were both using the term 'background.js'.

Reply:
Have you given this any thought?

Reply:
Will do.